.github/workflows/pull-request.yml

```diff
@@ -53,13 +54,15 @@ jobs:
 
       - name: Get all urls from sitemap.xml
         id: parse-sitemap
+        if: ${{ github.actor != 'dependabot[bot]' }}
         uses: cobraz/parse-sitemap@main
         with:
           sitemap-url: ${{ steps.firebase.outputs.details_url }}/sitemap/sitemap-index.xml
           replace-domain: https://bjerk.io|${{ steps.firebase.outputs.details_url }}
 
       - name: Run Lighthouse
         uses: treosh/lighthouse-ci-action@v7
+        if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
Notice that I even added Lighthouse-testing. Not only because we don't have a public URL, but also because it seems _to much_ to run Lighthouse tests every dependency bump.